mslib/msui/qt5/ui_tableview_window.py

```diff
@@ -88,6 +91,17 @@ def retranslateUi(self, TableViewWindow):
         self.btCloneWaypoint.setText(_translate("TableViewWindow", "clone"))
         self.btDeleteWayPoint.setText(_translate("TableViewWindow", "delete selected"))
         self.btInvertDirection.setText(_translate("TableViewWindow", "reverse"))
+        self.btRoundtrip.setText(_translate("TableViewWindow", "make roundtrip"))
         self.actionCloseWindow.setText(_translate("TableViewWindow", "Close Window"))
         self.actionCloseWindow.setShortcut(_translate("TableViewWindow", "Ctrl+W"))
 
+
```

Comment:
if this part is not needed please remove it. I guess it is manually added?

Reply:
May, this may not be apparent. These files in the qt5 directory are "automatically" generated from the ui files in the "ui" directory. We use the qt-creator thingie to modify the .ui files and use "pyuic5" to convert the .ui file to a .py file (e.g. `pyuic5 mslib/msui/ui/ui_mainwindow.ui > mslib/msui/qt5/ui_mainwindow.py`)

Reply:
It might be sensible to actually generate them automatically with python setup.py build, but we are not doing that, currently.

Reply:
If you did use the creator, you should also add the changes to the ui files to the merge request.

Reply:
These were created via pyuic5, I seem to have accidentally added an option which generated more code than necessary.
The changed ui files are in this commit as well, already.